test/jquery.ime.test.fixtures.js

```diff
@@ -2211,6 +2211,38 @@ var palochkaVariants = {
 			{ input: 'a~/e~\\i~^o~-u~v', output: 'áèîōǔ', description: 'Ibibio tilde áèîōǔ' }
 		]
 	},
+	{
+		description: 'Jawi test',
+		inputmethod: 'id-jawi',
+		tests: [
```

Comment:
Would be nice to have test cases for the more complex rules.

Reply:
On it